README.md

```diff
@@ -60,7 +60,11 @@ When in doubt, start on the [mailing-list](#mailing-list).
 
 ### Weekly Call
 
-The contributors and maintainers of all OCI projects have a weekly meeting Wednesdays at 2:00 PM (USA Pacific).
+The contributors and maintainers of all OCI projects have a weekly meeting on Wednesdays at:
+
+* 9:00 AM (USA Pacific), during [even weeks][iso-week].
```

Comment:
thanks for chiming in, cross-reference: https://groups.google.com/a/opencontainers.org/d/msg/dev/p0mTOspVgd0/ULXnARy9BAAJ